Cargo.toml

```
@@ -9,14 +9,28 @@ readme = "readme.md"
 license = "MIT"
 keywords = ["pathfinding", "dijkstra", "a-star", "grid"]
 categories = ["algorithms"]
-exclude = ["/.vscode/*", "/todo.md", "/.gitignore", "/.github/*", "/benchmark/*", "/img/*"]
+exclude = [
+    "/.vscode/*",
+    "/todo.md",
+    "/.gitignore",
+    "/.github/*",
+    "/benchmark/*",
+    "/img/*",
+]
 
 [dependencies]
-fnv = "1.0"
-rayon = { version = "1.5", optional = true }
+hashbrown = { version="0.11.2", features=["rayon"] }
```

```suggestion
hashbrown = "0.11.2"
```
`features=["rayon"]` breaks code that doesn't want the rayon-feature. See features suggestion below for alternative